app/src/main/res/values-ja/strings.xml

```diff
@@ -0,0 +1,146 @@
+    <string name="certificate_error_title">証明書を追加できませんでした。</string>
+    <string name="dlg_err_connection_failed">接続失敗</string>
+    <string name="dlg_err_failed_unknown_host">不明なアドレス：％s</string>
+    <string name="dlg_err_failed_wrong_http_credentials">無効なHTTP認証</string>
+    <string name="dlg_err_failed_timeout">サーバーが応答しません（タイムアウト）。</string>
+    <string name="dlg_err_failed_connect_failure">不明な接続失敗</string>
+    <string name="dlg_err_failed_not_found">無効なスケジュールURL</string>
+    <string name="dlg_err_failed_parse_failure">応答を解析できませんでした</string>
+    <string name="dlg_err_failed_ssl_failure">SSL障害</string>
+    <string name="dlg_err_failed_http_cleartext_not_permitted">HTTPは許可されていません。 HTTPSを使用してください。</string>
+    <string name="uptodate">スケジュールは最新です。</string>
+    <string name="schedule_changes_dialog_title">スケジュールを更新</string>
+    <string name="schedule_changes_dialog_updated_to_text">スケジュールが更新されました。</string>
+    <string name="schedule_changes_dialog_changed_new_cancelled_text">. ここに変更があります<xliff:g example="3 lectures" id="num_changes">%1$s</xliff:g>, <xliff:g example="2 are" id="num_new">%2$s</xliff:g> 新規, <xliff:g example="1 is" id="num_canceled">%3$s</xliff:g> キャンセルした.</string>
+    <string name="schedule_changes_dialog_affected_text">変更は、お気に入りの講義の<xliff:g example="2" id="num_marked">%1$d</xliff:g>に影響します。</string>
+    <string name="schedule_changes_dialog_browse">ブラウズ</string>
+    <string name="schedule_changes_dialog_later">後で</string>
+    <string name="progress_loading_data">&#8230;スケジュールを取得</string>
+    <string name="progress_processing_data">&#8230;スケジュールを処理中</string>
+    <string name="load_data">ロード</string>
+    <string name="choose_day">日付を選択</string>
+    <string name="day">日付</string>
+    <string name="fahrplan">スケジュール</string>
+    <string name="appVersion">アプリバージョン</string>
+    <string name="schedule_updated_to">"         <xliff:g example="Mildenberg 2018-01-16 16:50" id="version">%s</xliff:g>に更新しました。"</string>
+    <string name="schedule_updated">スケジュールが更新されました。</string>
+    <string name="settings">設定</string>
+    <string name="session_details_section_title_session_online">オンラインイベント</string>
+    <string name="session_details_section_title_links">リンク</string>
+    <string name="choose_alarm_time">アラーム時間を選択</string>
+    <string name="reminders">アラーム</string>
+    <string name="usage">注：アラームを設定するには、講義を長押しします。</string>
+    <string name="libraries">" \t\tこのアプリは次のライブラリを使用しています。T\t\\tKotlin, Google Support Library, BetterLinkMovementMethod, Email Intent Builder, \t\tEngelsystem, Espresso, OkHttp, SnackEngage, TraceDroid \t"</string>
+    <string name="about_data_privacy_statement">"         <![CDATA[<a href="https://github.com/EventFahrplan/EventFahrplan/blob/master/DATA-PRIVACY-DE.md">データプライバシーステートメント（ドイツ語）</a>]]>     "</string>
+    <string name="issues">" \t\t<![CDATA[<a href="https://github.com/EventFahrplan/EventFahrplan/issues">問題または機能のリクエスト</a>]]> \t"</string>
```

Comment:
Please remove the surrounding quotes and whitespace or tabs. It might have been added by the editor.

Reply:
Thanks for removing the quotes. The tabs (`\t`) and whitespace can be removed, too.